src/main/features/dashboard/views/claimant.njk

```diff
@@ -1,5 +1,5 @@
 {% extends "layout.njk" %}
-{% from "responseStatus.njk" import responseStatusForClaimant %}
+{% from "main/features/dashboard/views/macros/status.njk" import statusForClaimant, claimantRespondToOffer %}
```

Comment:
I would have thought that `macros/status` would work as it should be in the search path

Reply:
updating now